.github/workflows/increment-plugin-versions.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           - '2.5'
           - 3.x
           - '3.0'
+          - 1.x
```

Comment:
@prudhvigodithi Seems like our workflow will keep adding this `1.x` branch for version increment through [here](https://github.com/opensearch-project/opensearch-build/blob/3a0fa7a0e82d22b5ed3fa7474bcab33e9db177c1/src/manifests_workflow/input_manifests.py#L188)
although we manually removed it before. https://github.com/opensearch-project/opensearch-build/pull/3022